src/main/java/net/imagej/maven/AbstractCopyJarsMojo.java

```diff
@@ -54,15 +42,21 @@
 import org.apache.maven.plugin.AbstractMojo;
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.project.MavenProject;
-import org.codehaus.plexus.interpolation.EnvarBasedValueSource;
-import org.codehaus.plexus.interpolation.ObjectBasedValueSource;
-import org.codehaus.plexus.interpolation.PrefixAwareRecursionInterceptor;
-import org.codehaus.plexus.interpolation.PrefixedValueSourceWrapper;
-import org.codehaus.plexus.interpolation.PropertiesBasedValueSource;
-import org.codehaus.plexus.interpolation.RecursionInterceptor;
-import org.codehaus.plexus.interpolation.RegexBasedInterpolator;
+import org.codehaus.plexus.interpolation.*;
```

Comment:
We don't use wildcard imports in ImageJ.

For the reasons, see:
* https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad
* http://forum.imagej.net/t/import-package-rather-than-class/491/3?u=imagejan